libcaliptra/src/caliptra_fuses.h

```diff
@@ -0,0 +1,20 @@
+// Licensed under the Apache-2.0 license
+
+#pragma once
+
+#include "caliptra_api.h"
+
+// WARNING: THOSE APIS ARE INTENTED FOR SIMULATION ONLY.
```

Comment:
Spelling: "INTENTED"

Reply:
Fixed, but I need to discuss with Fabrizio what the long-term status of these functions will be.